### PR TITLE
Datetime localization for Azerbaijani language

### PR DESCRIPTION
**Pull request by EmiXLabs**
Added translation for absent language names and datetime strings (moved to Transifex),
Added datetime format for Azerbaijani locale

### CODE REVIEW

File: django/conf/locale/az/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -907,7 +909,7 @@ msgstr "%s MB"
 #: template/defaultfilters.py:902
 #, python-format
 msgid "%s GB"
-msgstr "%s GB"
+msgstr "%s QB"
```
Comment (EmiXLabs) [inline django/conf/locale/az/LC_MESSAGES/django.po]: in Azerbaijani: Qiqabayt - QB

File: django/conf/locale/az/formats.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    '%d.%m.%Y %H:%M:%S.%f',  # '25.10.2006 14:30:59.000200'
+    '%d.%m.%Y %H:%M',        # '25.10.2006 14:30'
+    '%d.%m.%Y',              # '25.10.2006'
+    '%d.%m.%y %H:%M:%S',     # '25.10.06 14:30:59'
+    '%d.%m.%y %H:%M:%S.%f',  # '25.10.06 14:30:59.000200'
+    '%d.%m.%y %H:%M',        # '25.10.06 14:30'
+    '%d.%m.%y',              # '25.10.06'
+]
+DECIMAL_SEPARATOR = '.'
+THOUSAND_SEPARATOR = ','
```
Comment (EmiXLabs) [inline django/conf/locale/az/formats.py]: 12,377.34

**Comment by timgraham** [comment]
Hi, thanks for submitting this.  Translations are handled at Transifex. See https://docs.djangoproject.com/en/dev/internals/contributing/localizing/#translations. This PR should only contain the Python files. Do you have a colleague who can +1 your work in `formats.py`?

**Comment by EmiXLabs** [comment]
@jafarovr, @seferov, @samadli - Can you guys check formats.py file? Used same settings as in ubuntu and windows os locale settings for Azerbaijani.

**Comment by seferov** [comment]
@timgraham @EmiXLabs +1

**Comment by jafarovr** [comment]
@timgraham @EmiXLabs +1

File: django/conf/locale/az/formats.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    '%d.%m.%Y %H:%M:%S.%f',  # '25.10.2006 14:30:59.000200'
+    '%d.%m.%Y %H:%M',        # '25.10.2006 14:30'
+    '%d.%m.%Y',              # '25.10.2006'
+    '%d.%m.%y %H:%M:%S',     # '25.10.06 14:30:59'
+    '%d.%m.%y %H:%M:%S.%f',  # '25.10.06 14:30:59.000200'
+    '%d.%m.%y %H:%M',        # '25.10.06 14:30'
+    '%d.%m.%y',              # '25.10.06'
+]
+DECIMAL_SEPARATOR = ','
+THOUSAND_SEPARATOR = '\xa0'  # non-breaking space
```
Comment (EmiXLabs) [inline django/conf/locale/az/formats.py]: 12 345,45 - Decimal separator officially is ',' symbol

**Comment by timgraham** [comment]
merged in dda2a3cf4cc29d01de180f66d19441f300732e52, thanks!

**Pull request closed without merging**